src/witan/models/dem/ccm/components_functions.clj

```diff
@@ -97,7 +97,30 @@
                                   (wds/safe-divide [(- national-trend last-year-nt) last-year-nt]))))
       (ds/select-columns [:sex :age :year :prop-diff])))
 
-(defn apply-national-trend
+
+;;recursive instead of loop-recur
+(defn apply-national-trend-to-data
```

Comment:
sorry, this was a failed reduce attempt that I forgot to remove- have taken it out now!